composer.json

```
@@ -14,7 +14,7 @@
     "php": "^8.2",
     "ext-json": "*",
     "ext-sockets": "*",
-    "datadog/dd-trace": "^0.87 || ^0.88 || ^0.89 || ^0.90",
+    "datadog/dd-trace": "^0.87 || ^0.88 || ^0.89 || ^0.90 || ^0.91",
```

This is getting unwieldy, maybe start chaging this to a range. I am not sure if dependabot understands this compatibility range though.

```suggestion
    "datadog/dd-trace": ">=0.87 <0.92",
```